index.ts

```diff
@@ -0,0 +1,12 @@
+// Default export of D2Api
+import D2Api from "./src/api/d2-api";
```

Comment:
Not sure about the file names:

I find the d2-api-metadata and d2-api-models to be reduntant. I guess metadata and models would suffice.

Reply:
Good point, renamed.